docs/run_in_existing_vpc.md

```diff
@@ -45,6 +45,19 @@ kops update cluster ${CLUSTER_NAME}
 Review the changes to make sure they are OK -  the Kubernetes settings might not be ones you want on a shared VPC (in which case,
 open an issue!)
 
+There is currently a [bug[(https://github.com/kubernetes/kops/issues/476) where kops will tell you that it will modify the VPC and InternetGateway names:
```

Comment:
I don't believe that fixes this. I have a note that says it is related to https://github.com/kubernetes/kops/issues/192, which I think @justinsb mentioned.

Not sure how I feel about adding TODO items in user facing docs. How about we add a note in the bug instead?